localisation/00_zuwc_united_localisation.csv

```diff
@@ -977,14 +977,14 @@ turn_yourself_into_monstrous_undead;Turn yourself into [Root.GetMonstrousUndeadN
 EVTOPTA_WCDAC_5005;I lost my clan...;;Ich habe meinen Clan verloren...;;;;;;;;;;;x
-EVTDESC_WCDAC_5010;As soon as the first stars began to light up in the sky of Azeroth, you collected all the pale-skinned of your clan from a large clan fire. They told you that the Hour of Twilight is already very close and it will really happen on this planet. About this, pale-skinned told you about some creatures of the abyss, the Old Gods, who were locked up by the Titans in magical prisons deep beneath the earth's crust of this world. This is very strange to you, but it's all very similar to the truth. If the Old Gods and the Void really exist, then who are we to protest their will?;;Kaum dass die ersten Sterne den Himmel Azeroths zu leuchten begannen, versammelt Ihr alle hellh�utigen Mitglieder Eures Clans am gro�en Lagerfeuer. Sie verk�nden Euch, dass die Stunde des Zwielichts bereits sehr nah ist und dass es in dieser Welt geschehen wird. Von einigen der Gesch�pfe der Leere, den alten G�ttern, die von den Titanen in magischen Gef�ngnissen tief unter der Erdkruste eingekerkert wurden, erz�hlen sie Euch. Das erscheint Euch sehr seltsam, aber es scheint der Wahrheit sehr nahe zu kommen. Wenn die alten G�tter und die Leere wirklich existieren, wer sind wir uns Ihrem Willen entgegenzustellen?;;;;;;;;;;;x
+EVTDESC_WCDAC_5010;As soon as the first stars began to light up in the sky of Azeroth, you collected all the pale-skinned of your clan from a large clan fire.\n\nThey told you that the Hour of Twilight is already very close and it will really happen on this planet. About this, pale-skinned told you about some creatures of the abyss, the Old Gods, who were locked up by the Titans in magical prisons deep beneath the earth's crust of this world.\n\nThis is very strange to you, but it's all very similar to the truth. If the Old Gods and the Void really exist, then who are we to protest their will?;;Kaum, dass die ersten Sterne den Himmel Azeroths zu leuchten begannen, versammelt Ihr alle hellh�utigen Mitglieder Eures Clans am gro�en Lagerfeuer.\n\nSie verk�nden Euch, dass die Stunde des Zwielichts bereits sehr nah ist und dass es in dieser Welt geschehen wird. Von einigen der Gesch�pfe der Leere, den alten G�ttern, die von den Titanen in magischen Gef�ngnissen tief unter der Erdkruste eingekerkert wurden, erz�hlen sie Euch.\n\nDas erscheint Euch sehr seltsam, aber es kommt der Wahrheit wohl sehr nahe. Wenn die alten G�tter und die Leere wirklich existieren, wer sind wir uns ihrem Willen entgegenzustellen?;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCDAC_5010;As soon as the first stars began to light up in the sky of Azeroth, you collected all the pale-skinned of your clan at a large firepit.\n\nThey informed you that the Hour of Twilight is close and that it will happen on this planet. The pale-skinned orcs told you about some creatures of the abyss, the Old Gods, who were locked up by the Titans in magical prisons deep beneath the earth's crust of this world.\n\nThis is very strange to you, but it's all very similar to the truth. If the Old Gods and the Void really exist, then who are we to protest their will?;;Kaum, dass die ersten Sterne den Himmel Azeroths zu leuchten begannen, versammelt Ihr alle hellh�utigen Mitglieder Eures Clans am gro�en Lagerfeuer.\n\nSie verk�nden Euch, dass die Stunde des Zwielichts bereits sehr nah ist und dass es in dieser Welt geschehen wird. Von einigen der Gesch�pfe der Leere, den alten G�ttern, die von den Titanen in magischen Gef�ngnissen tief unter der Erdkruste eingekerkert wurden, erz�hlen sie Euch.\n\nDas erscheint Euch sehr seltsam, aber es kommt der Wahrheit wohl sehr nahe. Wenn die alten G�tter und die Leere wirklich existieren, wer sind wir uns ihrem Willen entgegenzustellen?;;;;;;;;;;;x
```